packages/web/src/components/wip-ramp/commandersList.tsx

```diff
@@ -50,7 +49,7 @@ export const CommandersList = () => {
                         <List
                             itemLayout="horizontal"
                             dataSource={myData}
-                            renderItem={(item: Record<string, any>) => (
+                            renderItem={(item) => (
```

Comment:
Jelikoz mame datovy typ definovany uz nemusim znovu definovat tady, Typescript presne vi jake tam jsou parametry + s tim definovanym datovym typem ti to bude presne napovidat ty fields. Takze se ti uz nestane ze si omylem napsat NAME misto commanderName